ios/fotos/Info.plist

```diff
@@ -35,6 +35,8 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSMicrophoneUsageDescription</key>
```

Comment:
@emadbaqeri  Is it mandatory to get Microphone permission to play a video?

Reply:
Hi @ghorbani-m, according to [the documentation](https://github.com/expo/expo/tree/sdk-46/packages/expo-av#installation-in-bare-react-native-projects), yeah we have to do that.